aws/data_aws_bucket_policy.go

```diff
@@ -36,6 +36,11 @@ func DataAwsBucketPolicy() *schema.Resource {
 						Principal: map[string]string{
 							"AWS": fmt.Sprintf("arn:aws:iam::%s:root", d.Get("databricks_account_id").(string)),
 						},
+						Condition: map[string]map[string]string{
+							"StringEquals": {
+								"aws:PrincipalTag/DatabricksAccountId": d.Get("databricks_account_id").(string),
```

Comment:
> Get your databricks-account-id. Go to the [account console](https://protect-us.mimecast.com/s/GK-NCOY0xwuNywrCER2IH?domain=docs.databricks.com) and click the in the bottom left corner. Under Account ID, select and copy the ID.

documentation meant the `E2 account id`, but in this data resource the `databricks_account_id` is AWS Account ID of Databricks